projects/currency-converter.md

```diff
@@ -22,62 +22,55 @@ It is expected that your app will have at least these features and functionality
 
 ### `JS2 week1` - Browser environment, DOM manipulation, DOM event listeners
 
-- create a form to insert a new currency rate.
+- Create a form to insert a new currency rate.
 
-- implement one of the two (or both!)
+- Implement one of the two (or both!)
```

Comment:
```suggestion
// TODO: describe the task to implement currency conversion. 
- Implement one of the two (or both!)
```
At this point we have not asked to implement the currency conversion yet, so I suppose "Update existing currency conversion with a new rate" falls out of context if I understand it correctly.

Reply:
Good point. Added an explicit ask about the conversion